R/data.table.R

```diff
@@ -1,6 +1,6 @@
 dim.data.table = function(x)
 {
-  if (!cedta()) return(NextMethod()) # nocov
+  if (!cedta(verbose=FALSE)) return(NextMethod()) # nocov
```

Comment:
What's the exact reason to forcing this to F here? Why in this particular method we want to see less?

Reply:
The reason is that we call dim in every print statement which are often not cedta

Reply:
Isn't that resolved by switching to `nrow(x)[1L]` in `print.data.table`?